### PR TITLE
Overview button change 2

### PR DESCRIPTION
**Pull request by fpsbernardo**

### CODE REVIEW

File: src/overview/button/VerificationButton.js [hunk 1]
```diff
@@ -4,20 +4,26 @@ import React from 'react';
 import styled from 'styled-components';
 import { GradientButton, Link } from '../../ui';
 import { VERIFICATION_INTRO_ROUTE } from '../../verification/constants';
-import svgRocket from './img/rocket.svg';
+import bitcoin from './img/bitcoin.png';
 
 const Rocket = styled.img`
```
Comment (v3rm0n) [inline src/overview/button/VerificationButton.js]: Not a Rocket anymore

File: src/overview/Overview.spec.js [hunk 2]
```diff
@@ -36,17 +38,22 @@ describe('Overview component', () => {
   });
 
   it('renders verification button for not verified users', () => {
-    expect(component.contains(<VerificationButton />)).toBe(true);
+    expect(component.contains(<BuyButton to={VERIFICATION_INTRO_ROUTE} />)).toBe(true);
   });
 
-  it('does not render verification button for verified users', () => {
+  it('renders verification button for verified users when balance is 0', () => {
```
Comment (JordanValdma) [inline src/overview/Overview.spec.js]: renders what button?

File: src/overview/Overview.spec.js [hunk 3]
```diff
@@ -36,17 +38,22 @@ describe('Overview component', () => {
...
     component.setProps({ isVerified: true });
-    expect(component.contains(<VerificationButton />)).toBe(false);
+    expect(component.contains(<BuyButton to={routes.ORDER} />)).toBe(true);
+  });
+
+  it('does not render verification button for verified users when balance is not 0', () => {
```
Comment (JordanValdma) [inline src/overview/Overview.spec.js]: which button?

File: src/overview/Overview.spec.js [hunk 4]
```diff
@@ -36,17 +38,22 @@ describe('Overview component', () => {
...
+    component.setProps({ isVerified: true, balance: 10 });
+    expect(component.contains(<BuyButton to={routes.ORDER} />)).toBe(false);
   });
 
   it('does not render verification button for users with verification status "verification pending"', () => {
```
Comment (JordanValdma) [inline src/overview/Overview.spec.js]: what button?

**Comment by fpsbernardo** [comment]
@JordanValdma Let's not merge this for now. User testing seems to point out that the "Buy Bitcoin" is confusing so I suggest keeping the "Verify account" button for now.

**Comment by jayrmotta** [comment]
@fpsbernardo @JordanValdma 

Shall we close this PR?

**Pull request closed without merging**